arches_references/src/arches_references/constants.ts

```diff
@@ -44,3 +43,13 @@ export const NOTE_CHOICES = {
     note: "note",
     description: "description",
 };
+
+// Temporary workaround until received from backend
+export const ENGLISH = {
```

Comment:
++. IMO the lack of a ?Language? endpoint is starting to affect a few different places and should be prioritized somewhat soon 👍